lib/error.g

```diff
@@ -184,26 +185,77 @@ BIND_GLOBAL("ErrorInner",
         LEAVE_ALL_NAMESPACES();
         JUMP_TO_CATCH(1);
     fi;
-        
+
+    # Local functions that print the user feedback.
+    printEarlyMessage := function(stream, earlyMessage)
```

Comment:
You don't need to pass `earlyMessage` as an argument, it is available automatically in this closure. I.e. just drop the second argument here and in calls, no other change, and things will work just fine.